mlir-sycl/lib/Dialect/IR/SYCLOpsDialect.cpp

```diff
@@ -56,6 +56,9 @@ mlir::sycl::SYCLDialect::parseType(mlir::DialectAsmParser &Parser) const {
   if (Keyword == "accessor_impl_device") {
     return mlir::sycl::AccessorImplDeviceType::parseType(Parser);
   }
+  if (Keyword == "accessor_subscript") {
+    return mlir::sycl::AccessorSubscriptType::parseType(Parser);
+  }
```

Comment:
{} not necessary, consider changing all the other if stmts

Reply:
@victor-eds Does the refactor code you working on includes removing braces here? If not, I will do it here.

Reply:
I will wait for Victor's PR, then modify the code here if needed.

Reply:
No need to wait for my PR. As it's a heavy refactoring, I can take it. No worries.

Reply:
Thanks Victor 😃